examples/03_connectivity/plot_signal_extraction.py

```diff
@@ -73,9 +74,85 @@
 plotting.plot_matrix(correlation_matrix, figure=(10, 8), labels=labels[1:],
                      vmax=0.8, vmin=-0.8, reorder=True)
 
+##############################################################################
+# Load confounds from file using a flexible strategy with load_confounds
+# ----------------------------------------------------------------------
+# The :mod:`nilearn.load_confounds` module can be used to create
+# a :class:`nilearn.load_confounds.Confounds` class with flexible
```

Comment:
The doc build is currently failing because these refs are broken. We should replace them with ":func:`nilearn.load_confounds.load_confounds`", and update the text.